benchmarks/scala-stm/src/main/scala/org/renaissance/scala/stm/Philosophers.scala

```diff
@@ -17,7 +17,7 @@ class Philosophers extends RenaissanceBenchmark {
   // TODO: Consolidate benchmark parameters across the suite.
   //  See: https://github.com/renaissance-benchmarks/renaissance/issues/27
 
-  private val THREAD_COUNT = Runtime.getRuntime.availableProcessors
+  private val THREAD_COUNT = 8
```

Comment:
Maybe the default should be `math.min(8, Runtime.getRuntime.availableProcessors`?

Reply:
Why not. I can change this.
I like this strategy for the rework that will be necessary for #27 .